src/Idris/Elab/Interface.idr

```diff
@@ -238,6 +232,20 @@ getMethToplevel {vars} env vis iname cname constraints allmeths params sig
     methName : Name -> Name
     methName n = UN (Basic $ bindName n)
 
+    collectImplicitNames : RawImp -> List Name
+    collectImplicitNames (IPi _ _ Explicit _        _ ty) = []
+    collectImplicitNames (IPi _ _ _        (Just n) _ ty) = n :: collectImplicitNames ty
+    collectImplicitNames (IPi _ _ _        Nothing  _ ty) = collectImplicitNames ty
```

Comment:
Skipping implicit arguments without names can cause an error, but I'm not sure how to handle them properly

Reply:
Example with an error:
```idris
interface Iface where
  method : {_ : Type} -> Type
```

Reply:
As far as I know, an implicit argument without a name cannot be passed. It might be useful to add a “disable implicits” mode as [suggested in Discord](https://discord.com/channels/827106007712661524/827109161857712128/1329577616641888340) and use it when elaboring. It could also be useful in user code